CLAUDE.md

```
@@ -0,0 +1,167 @@
+./gradlew jacocoTestReport
+
+# 로컬 인프라 실행 (MySQL, Redis master/replica, Kafka, Kafka-UI)
+docker-compose -f ./docker/infra-compose.yml up
+
+# 모니터링 실행 (Prometheus + Grafana — http://localhost:3000, admin/admin)
+docker-compose -f ./docker/monitoring-compose.yml up
+```
+
+## Multi-Module Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)